modules/EnsEMBL/Web/Document/Image/R2R.pm

```diff
@@ -53,8 +53,14 @@ sub find_ss_in_compara {
 
     my $gma = $database->get_GeneMemberAdaptor();
```

Comment:
This is not being used so can be removed.

Reply:
Done!